src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -141,7 +145,7 @@ public Test BuildFrom(IMethodInfo method, Test parentSuite)
                     tests.Add(test);
             }
 
-            return tests.Count > 0
+            return shouldParameterizeForFewerThanTwoTests || tests.Count > 0
```

Comment:
This seems slightly complicated to me, since I'm still thinking that checking

```C#
if (tests.Count > 0 && builders.Count > 0)
...
```
won't do the job.